src/esbuild_plugins/build_routes/get_route_html.ts

```diff
@@ -1,40 +1,61 @@
 import { normalize } from "https://deno.land/std@0.177.0/path/mod.ts";
-import { join } from "https://deno.land/std@0.182.0/path/mod.ts";
 import { format } from "npm:prettier";
 
-export const get_route_html = async ({ html, css, base_path, site_dir }: {
+const template = (styles: string, hydrate: string, html: string) =>
+	`<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Svelte + Deno</title>
+
+    <style>${styles}</style>
+    <script type="module">${hydrate}</script>
```

Comment:
```suggestion
    <script type="module">${hydrate}</script>
    <style>${styles}</style>
```
shodln't these be the other way round, so the browser know's there nothing that could change the styles?

Reply:
Modules are asynchronous by default, but sure!

Reply:
ah yeah course